lib/onebox/sanitize_config.rb

```diff
@@ -12,7 +12,7 @@ module Config
         'embed'  => %w[height src type width],
         'iframe' => %w[allowfullscreen frameborder height scrolling src width],
         'source' => %w[src type],
-        'video'  => %w[controls height loop width],
+        'video'  => %w[controls height loop width autoplay],
```

Comment:
Are we sure we want to allow "`autoplay`"?

Reply:
Actually yes!

Gfycat is mostly a gif-like site. All videos are muted, and autoplay doesn't work on mobile.

One of the reasons we did our own onebox was to be able to autoplay only on the desktop.

We discussed this briefly here: https://meta.discourse.org/t/oneboxed-gfycat-slower-than-embedded-video/58313/8?u=falco